### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by Radheshyam1234**
Front-Quiz-Landing-Page1
Front-Quiz-Landing-Page2

### CODE REVIEW

File: src/themeData.ts [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        color: "white",
+        transform: "scale(0.98)",
+      },
+    },
+
+    //   actionBtnIcon: {
+    //     minW: "1.2rem",
+    //     h: "1.2rem",
+    //     my: "0.5rem",
+    //     color: "secondary.500",
```
Comment (Kothalkarpriya) [inline src/themeData.ts:120]: If commented code is not required you can remove it

**Review by Kothalkarpriya — APPROVED** [review]
Good work!

**Review by yashraj2312 — APPROVED** [review]
The codebase can be made cleaner by removing non-essential comments. Looks good to merge.

**Review by simran61 — APPROVED** [review]
Looks good
Clean code

**Pull request merged**